index.html

```diff
@@ -50,5 +51,237 @@ <h4>LET'S CONNECT</h4>
+      </div>
+      <div class="texts">
+        <h3 class="bold-text">CANOPY</h3>
+        <div class="dot"></div>
+        <p class="text-1">Back End Dev</p>
+        <div class="dot"></div>
+        <p class="text-2">2015</p>
+      </div>
+<div>
+  <p class="primary-text">
+    A daily selection of privately personalized reads; no accounts or
+    sign-ups required.
+  </p>
+</div>
+
+<div class="tags-div">
+<ul class="tags">
+  <li class="html">HTML</li>
+</ul>
+<ul class="tags">
+  <li class="css">CSS</li>
+</ul>
+<ul class="tags">
+  <li class="js">JavaScript</li>
+</ul>
+</div>
+<div class="project-div">
+  <button class="project">See Project</button>
+</div>
+    </section>
+
+<!--About myself-->
+    <section class="base">
+      <div class="about-myself">
+      <h1 class="about-base">About Myself</h1>
+      <p class="about-text">
+        Hello I'm a software developer! I can help you build a product, feature
+        a website. Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hesitate to contact me.
+      </p>
+      <h4 class="connect">LET'S CONNECT</h4>
+      
+        <ul class="connect-images">
+          <li><a href="#"></a> <img src="Assets\Vector.svg" alt="twitter" class="vector"/></a></li>
+          <li><a href="#"></a> <img src="Assets\Linkedin icon.svg" alt="Linkedin" class="vector"/></a></li>
+          <li><a href="#"></a> <img src="Assets\Vector (2).svg" alt="mail" class="vector"></a></li>
+          <li><a href="#"></a> <img src="Assets\Vector (1).svg" alt="github" class="vector"></a></li>
+          <li><a href="#"></a> <img src="Assets\Vector (3).svg" alt="hand" class="vector"></a></li>
+        </ul>
+
+      <div class="resume-div">
+        <button class="resume">Get my resume</button>
+      </div>
+
+      <!--languages-->
+      <div class="languages">
+        <p>Languages</p>
+      </div>
+
+      <div class="language-bar">
+        <ul class="tags">
+          <img src="Assets\Ellipse 1@2x.svg" alt="java" class="java" />
+          <li class="js">JavaScript</li>
+        </ul>
+        <ul class="tags">
+          <img src="Assets\Ellipse 1h@2x.svg" alt="html" class="html" />
+          <li class="html">HTML</li>
+        </ul>
+        <ul class="tags">
+          <img src="Assets\Ellipse 1c@2x.svg" alt="css" class="css" />
+          <li class="css">CSS</li>
+        </ul>
+      </div>
+
+      <div>
+        <div class="frame">
+          <p class="frameworks">Frameworks</p>
+          <img
+            src="Assets\navigate_next_FILL0_wght400_GRAD0_opsz48.png"
+            alt="vector"
+            class="forward"
+          />
+        </div>
+
+        <div class="frame">
+          <p class="skills">Skills</p>
+          <img
+            src="Assets\navigate_next_FILL0_wght400_GRAD0_opsz48.png"
+            alt="vector"
+            class="forward"
+          />
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Please 🙏🏽 correct the margin, padding, align content, background color, narrow-bottom and the size of the narrows to be similar to the Figma design 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/58642949/7141cff3-0540-4a67-ad6c-93d17f21557e) |![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/58642949/a7dc8359-5522-4459-9d60-a773d1219275) |